### PR TITLE
LPS-131866 | master

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-131866

This is only for master.

@david-gutierrez-mesa

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentUI.testcase [hunk 1]
```diff
@@ -231,4 +231,66 @@ definition {
...
+
+		task ("Add an expired web content based on Basic Web Content") {
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Test Site Name",
+				title = "Expired WebContent Title");
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContent.expireCP(webContentTitleList = "Expired WebContent Title");
```
Comment (david-gutierrez-mesa) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentUI.testcase:260]: We should find a way to do this with JSON

**Comment by david-gutierrez-mesa** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentUI.testcase (hunk 1 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentUI.testcase:260]: I tried. It seems like json ws doesn’t support that. Anyway, I will confirm with our devs.

**Comment by Tim-Cao** [comment]
Hi @david-gutierrez-mesa
See https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/service/ServiceContext.java#L1514
https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowConstants.java#L23
WorkflowAction only supports draft and publish.

**Pull request closed without merging**